config/config-server/resources/cruise-config.xsd

```diff
@@ -712,7 +712,7 @@
     </xsd:simpleType>
     <xsd:simpleType name="labelType">
         <xsd:restriction base="xsd:string">
-            <xsd:pattern value="(([a-zA-Z0-9_\-.!~*'()#:])*[$#]\{[a-zA-Z0-9_\-.!~*'()#:]+\}([a-zA-Z0-9_\-.!~*'()#:])*)+"/>
+            <xsd:pattern value="(([a-zA-Z0-9_\-.!~*'()#:])*[$#]\{[a-zA-Z0-9_\-.!~*'()#:]+(\[:(\d+)\])?\}([a-zA-Z0-9_\-.!~*'()#:])*)+"/>
```

Comment:
added test in commit [bf94e39
](https://github.com/alexschwartz/gocd/commit/bf94e39f958e36cf7639c6ffc82efd104738c720)